app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    old_file, new_file = command.split(" ")[1:]
+
+    if old_file != new_file:
```

Comment:
Putting the function's main logic inside an `if` condition is usually considered a bad practice.

Also, the function shouldn't copy file if the command isn't `cp`.